Makefile

```diff
@@ -35,7 +35,7 @@ stop:
 	$(DOCKER_COMPOSE) down -v
 
 checks:
-	$(DOCKER_COMPOSE) run --rm web /bin/bash -c "\
+	$(DOCKER_COMPOSE) run --rm app /bin/bash -c "\
```

Comment:
At the top of this file there are some variables. One of them is `PROJECT_CODEBASE_DIR = "search/"`. We could make another called `PROJECT_NAME = "search"` and update the other to be `PROJECT_CODEBASE_DIR = "$(PROJECT_NAME)/"` or just create `SERVICE_NAME = "app"` and use this variable throughout the file?

Reply:
Yes, `SERVICE_NAME` changing is an indication it could be a variable.